### PR TITLE
Switch to the maintained pytest-lazy-fixtures package

### PR DESCRIPTION
**Pull request by mgorny**
`pytest-lazy-fixture` has not been maintained since 2022, and it [does not work properly with pytest-8 anymore](https://github.com/TvoroG/pytest-lazy-fixture/issues/65).  Switch to the maintained`pytest-lazy-fixtures` replacement.

-----

I've split poetry lock update to a separate commit, in case you preferred it done some other way.

### CODE REVIEW

File: tests/test_field.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import pytest
 from django.core.exceptions import ValidationError
-from pytest_lazyfixture import lazy_fixture
+from pytest_lazy_fixtures import lf
```
Comment (cclauss) [inline tests/test_field.py:3]: Given that `lf` is a bit cryptic (is it linefeed?) what about...
```suggestion
from pytest_lazy_fixtures import lf as lazy_fixture
```
and then revert all the remaining changes in this file.

File: tests/test_form_field.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import pytest
 from django import forms
-from pytest_lazyfixture import lazy_fixture
+from pytest_lazy_fixtures import lf
```
Comment (cclauss) [inline tests/test_form_field.py:3]:
```suggestion
from pytest_lazy_fixtures import lf as lazy_fixture
```
and then revert all the remaining changes in this file.

File: tests/test_model_form_field.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import pytest
-from pytest_lazyfixture import lazy_fixture
+from pytest_lazy_fixtures import lf
```
Comment (cclauss) [inline tests/test_model_form_field.py:2]:
```suggestion
from pytest_lazy_fixtures import lf as lazy_fixture
```
and then revert all the remaining changes in this file.

**Review by cclauss — COMMENTED** [review]
`from pytest_lazy_fixtures import lf as lazy_fixture` for readability.

**Comment by mgorny** [comment]
Sure, updated now.

**Comment by mfogel** [comment]
Thanks very much for the contribution!

Not sure why Github isn't running CI for this, but I ran the test locally and all looked good. Merging.

**Comment by mgorny** [comment]
Thanks!

**Pull request merged**